backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/crowd/service/CrowdAuthentication.kt

```diff
@@ -2,15 +2,15 @@ package com.tngtech.apicenter.backend.connector.crowd.service
 
 import com.atlassian.crowd.service.client.CrowdClient
 import com.tngtech.apicenter.backend.domain.entity.User
-import com.tngtech.apicenter.backend.domain.service.ExternalAuthenticationService
+import com.tngtech.apicenter.backend.domain.service.ExternalAuthentication
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
 import org.springframework.stereotype.Service
 import java.util.UUID
 
 @Service
 @ConditionalOnProperty(name = ["auth.service"], havingValue = "crowd")
-class CrowdAuthenticationService @Autowired constructor(private val crowdClient: CrowdClient) : ExternalAuthenticationService {
+class CrowdAuthentication @Autowired constructor(private val crowdClient: CrowdClient) : ExternalAuthentication {
```

Comment:
I'm for ditching "Service", but - at least in my opinion - class names should describe what they're doing (e.g. "Authenticator"). More "passive" names are reserved for data classes, value objects etc.
Does that make sense?

Reply:
ie. `*Persistence` to `*Persister`?

Reply:
Yes. Kind of. "Persister" feels unnatural, but it also feels awkward to call `Persistence.find()`. Maybe "repository", or "archive"? (Or even "vault"? ;))

Reply:
And I just noticed that "archive" and "vault" aren't that active, either. I think concreteness also plays a role here, as I can better imagine what an "archive" does than what "persistence" does.

As an aside, "persistence" would make - in my opinion - for a good package name, as these tend to describe the common theme, but nothing concrete like a class.